.github/workflows/dependabot-auto-merge.yml

```diff
@@ -0,0 +1,17 @@
+name: Dependabot auto-merge
+on: pull_request
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  dependabot:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Enable auto-merge for Dependabot PRs
+        run: gh pr merge --auto --merge "$PR_URL"
```

Comment:
Hmm, `--auto` is not allowed in our repository:

```
Message: ["Pull request Auto merge is not allowed for this repository"], Locations: [{Line:1 Column:72}]
```

What's your opinion on this feature, @juliangruber? Could you please enable it in the repository settings?

Reply:
We are also not using "merge commit" strategy in this repo, so I'll need to rework this line.

```suggestion
        run: gh pr merge --auto --squash "$PR_URL"
```

Reply:
Never mind, `--auto` works.

```
❯ gh pr merge --auto --squash https://github.com/filecoin-project/filecoin-station/pull/85
✓ Squashed and merged pull request #85 (deps(deps-dev): bump @typescript-eslint/eslint-plugin from 5.36.0 to 5.36.1)
```

Reply:
> Never mind, `--auto` works.

It works only if all required checks have already passed. When there are some checks in progress, we can the error message even with `--squash`.

```
❯ gh pr merge --auto --squash https://github.com/filecoin-project/filecoin-station/pull/86
Message: ["Pull request Auto merge is not allowed for this repository"], Locations: [{Line:1 Column:72}]
```

Reply:
<img width="874" alt="Screenshot 2022-09-01 at 10 08 05" src="https://user-images.githubusercontent.com/10247/187864925-81970306-f64e-4afe-ab70-45e5afa88e74.png">

I've just enabled this, is that enough?

Reply:
Yep, all seems to be good now.

```
❯ gh pr merge --auto --squash https://github.com/filecoin-project/filecoin-station/pull/92
✓ Pull request #92 will be automatically merged via squash when all requirements are met
```